### ISSUE TITLE
Create a data management command to copy data from the old mature tables to the new sensitive ones

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse#2627]
## Description

Create a Django data management command which will move data from the previous mature-related tables to the new sensitivity-related ones. Instructions for how to perform this change are present in the implementation plan associated with this work: https://docs.openverse.org/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.html#code (steps 4-5).

A deployment of the API will be necessary before handling downstream tickets.

## Additional context

Part of #383, blocked by #2626

**Comment by AetherUnbound** [comment]
Closing in favor of #3723 based on #3568.
